cs2_datatypes.h

```diff
@@ -377,6 +379,19 @@ class CNetworkVarChainer
 };
 static_assert(sizeof(CNetworkVarChainer) == 0x28, "Class didn't match expected size");
 
+// Size: 0x40
+class CPlayerPawnComponent
+{
+public:
+	uint8_t __pad0000[0x8]; // 0x0
+	// MNetworkDisable
+	// MNetworkChangeAccessorFieldPathIndex
+	CNetworkVarChainer __m_pChainEntity; // 0x8
+	CBasePlayerPawn* pawn;
```

Comment:
style note: i keep my pointers consistently like `void *ptr` instead of `void* ptr`.